# SPDX-FileCopyrightText: Copyright (c) 2025 Objectionary.com
# SPDX-License-Identifier: MIT
diff --git a/node_modules/grunt-mocha-cli/lib/index.js b/node_modules/grunt-mocha-cli/lib/index.js
index 59f7a07..2262b80 100644
--- a/node_modules/grunt-mocha-cli/lib/index.js
+++ b/node_modules/grunt-mocha-cli/lib/index.js
@@ -2,6 +2,7 @@
 const fs = require('fs')
 const path = require('path')
 const grunt = require('grunt')
+const os = require('os')
 
 const BOOL_OPTIONS = [
   'allow-uncaught',
@@ -40,7 +41,7 @@ const STRING_OPTIONS = [
   'timeout',
   'ui'
 ]
-const ARRAY_OPTIONS = ['globals']
+const ARRAY_OPTIONS = ['globals', 'node-option']
 
 module.exports = function(options) {
   return new Promise((resolve, reject) => {
@@ -59,6 +60,14 @@ module.exports = function(options) {
       spawnOptions.cmd += '.cmd'
     }
 
+    if (os.platform() === 'win32') {
+      spawnOptions.opts.shell = true
+    }
+
+    if (options['node-option'] && options['node-option'].length > 0) {
+      spawnOptions.opts.execArgv = options['node-option']
+    }
+
     if (!options.quiet && !options.save) {
       // Redirect the Mocha output directly to the terminal to maintain colors
       spawnOptions.opts.stdio = 'inherit'
